app/src/main/res/values/strings.xml

```diff
@@ -474,6 +474,8 @@
     <string name="alerter_notification_title">Notifications Available</string>
     <string name="alerter_notification_text">You have one or more unread notifications!</string>
 
+    <string name="alerter_notification_action_clear">Mark all as read</string>
```

Comment:
could also just be "Mark as read"

Reply:
I think it'd be better as "Mark as read".

Thoughts? @wax911

Reply:
I force pushed it to "Mark as read". I too think it's better that way.

Reply:
Agreed thanks!